components/accelerator/nvidia/peermem/component.go

```diff
@@ -98,56 +96,7 @@ const (
 )
 
 func (c *component) Events(ctx context.Context, since time.Time) ([]components.Event, error) {
-	dmesgC, err := components.GetComponent(dmesg.Name)
-	if err != nil {
-		return nil, err
-	}
-
-	var dmesgComponent *dmesg.Component
-	if o, ok := dmesgC.(interface{ Unwrap() interface{} }); ok {
-		if unwrapped, ok := o.Unwrap().(*dmesg.Component); ok {
-			dmesgComponent = unwrapped
-		} else {
-			return nil, fmt.Errorf("expected *dmesg.Component, got %T", dmesgC)
-		}
-	}
-	dmesgTailResults, err := dmesgComponent.TailScan()
-	if err != nil {
-		return nil, err
-	}
-
-	// dedup by minute level
-	seenMinute := make(map[int64]struct{})
-	events := make([]components.Event, 0)
-	for _, logItem := range dmesgTailResults.TailScanMatched {
-		if logItem.Error != nil {
-			continue
-		}
-		if logItem.Matched == nil {
-			continue
-		}
-		if logItem.Matched.Name != dmesg.EventNvidiaPeermemInvalidContext {
-			continue
-		}
-
-		minute := logItem.Time.Unix() / 60
-		if _, ok := seenMinute[minute]; ok {
-			continue
-		}
-		seenMinute[minute] = struct{}{}
-
```

Comment:
ignore it here. 

if name == dmesg.EventNvidiaPeermemInvalidContext {
    continue
    // some comment here.
}